### PR TITLE
Added a handler for Amazon

### LINKED ISSUE #2: Amazon Handler
- Create a handler for Amazon Jobs

### PR DESCRIPTION
**Pull request by sa-y-an**
- added a Handler for amazon 

Closes: #2

### CODE REVIEW

**Comment by sa-y-an** [comment]
@prince776 Please have a look !

File: src/handlers/AmazonHandler.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { Handler, Job } from "../types";
+import axios from 'axios';
+
+const url = 'https://www.amazon.jobs/en/search.json?radius=24km&facets%5B%5D=normalized_country_code&facets%5B%5D=normalized_state_name&facets%5B%5D=normalized_city_name&facets%5B%5D=location&facets%5B%5D=business_category&facets%5B%5D=category&facets%5B%5D=schedule_type_id&facets%5B%5D=employee_class&facets%5B%5D=normalized_location&facets%5B%5D=job_function_id&facets%5B%5D=is_manager&facets%5B%5D=is_intern&offset=0&result_limit=10&sort=relevant&latitude=&longitude=&loc_group_id=&loc_query=India&base_query=Software%20Development&city=&country=IND&region=&county=&query_options=&';
```
Comment (prince776) [inline src/handlers/AmazonHandler.ts]: `result_limit=10`
we're only getting 10 jobs, can you add pagination or just increase limit if that works.
I checked till limit of 100, and that works, but obviously, 100 jobs are not all relevant, please add some more filtering to it. This is the reason I didn't do it myself 

While post processing should remove plenty of these, maybe adding some time filtering to not consider job openings opened before 1 month can help.

**Review by prince776 — CHANGES_REQUESTED** [review]

File: src/handlers/AmazonHandler.ts (hunk 1 above)
Comment (sa-y-an) [reply src/handlers/AmazonHandler.ts]: - Sure, I checked the page, and there is no option for filtering on the API level based on the date 
- So, I am adding a filter after fetching the results!

**Comment by sa-y-an** [comment]
@prince776  Added a new commit, please check!

File: src/handlers/AmazonHandler.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        // filters out jobs > a month 
+        const jobs: any[] = rawJobs.filter( job => {
+            return (Date.now()- Date.parse(job.posted_date)) / 1000 <= 2592000;
+        } )
+
+        const result: Job[] = jobs.map(job => {
+            return {
+                id: job.id,
+                name: job.title,
+                url: job.job_path,
```
Comment (prince776) [inline src/handlers/AmazonHandler.ts]: `  url: '/en/jobs/2085280/software-development-engineer-tablet-software',`
we are getting this. please make it a complete url.

Comment (sa-y-an) [reply src/handlers/AmazonHandler.ts]: @prince776 added a commit for full URL !

**Review by prince776 — APPROVED** [review]

**Pull request merged**